x-pack/plugins/runtime_fields/README.md

```diff
@@ -4,11 +4,61 @@ Welcome to the home of the runtime field editor and everything related to runtim
 
 ## The runtime field editor
 
-The runtime field editor is exported in 2 flavours:
+### Integration
 
-* As the content of a `<EuiFlyout />`
+The recommended way to integrate the runtime fields editor is by adding a plugin dependency to the `"runtimeFields"` x-pack plugin. This way you will be able to lazy load the editor when it is required and it will not increment the bundle size of your plugin.
```

Comment:
I wonder if it is worth explaining here why we want to share the editor in this way. From what I can tell we are giving users a way to imperatively load a JSX element and I'm wondering what the benefits are of this approach vs a static component. Is it so that we can provide a way for plugins to lazy load the component? Happy if that is only reason, I just wanted to make sure I understood the intention correctly.

Reply:
So yes, the main reason would be the ability to lazy load. So if the editor depends on an external lib it should not increase the page load time.

The second reason is that it simplifies the API as it is not required anymore to provide core services to the editor. And any other plugin dependency is handled automatically instead of having to provide them all through props.